src/ui/public/parse_query/lib/from_user.js

```diff
@@ -1,42 +1,35 @@
 import _ from 'lodash';
-import { DecorateQueryProvider } from 'ui/courier/data_source/_decorate_query';
 
-export function ParseQueryLibFromUserProvider(es, Private) {
-  const decorateQuery = Private(DecorateQueryProvider);
+export function ParseQueryLibFromUserProvider() {
 
   /**
-   * Take text from the user and make it into a query object
-   * @param {text} user's query input
+   * Take userInput from the user and make it into a query object
```

Comment:
With the changes to this file, do we even need the [`toUser`](https://github.com/elastic/kibana/blob/db4ba51f2604b1414f505b42121ef04bb157f2e5/src/ui/public/parse_query/lib/to_user.js) module any more?